src/cpp/fastdds/domain/DomainParticipantFactory.cpp

```diff
@@ -386,6 +387,18 @@ ReturnCode_t DomainParticipantFactory::load_XML_profiles_string(
     return ReturnCode_t::RETCODE_OK;
 }
 
+ReturnCode_t DomainParticipantFactory::check_xml_static_discovery(
+    std::string& xml_file)
+{
+    XMLEndpointParser parser;
+    if (XMLP_ret::XML_ERROR == parser.loadXMLFile(xml_file))
```

Comment:
Right now, the parser only returns either error or ok, but that may change in the future. In that case, I'd only return ok whenever the parser does so.
```suggestion
    if (XMLP_ret::XML_OK != parser.loadXMLFile(xml_file))
```

Reply:
ok, committed